models/classes/preview/ItemPreviewerService.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2018 (original work) Open Assessment Technologies SA;
+ */
+
+namespace oat\taoItems\model\preview;
+
+use oat\oatbox\AbstractRegistry;
+use oat\oatbox\service\ConfigurableService;
+use oat\tao\model\ClientLibConfigRegistry;
+use oat\tao\model\modules\DynamicModule;
+
+/**
+ * Manage item previewers
+ *
+ * @author Jean-Sébastien Conan <jean-sebastien@taotesting.com>
+ */
+class ItemPreviewerService extends ConfigurableService
```

Comment:
I've got an issue with the fact that this class is juste a registry wrapper. Can we use it as a real configurable service with some real configuration instead of loading / saving stuff in a registry ?